core/server/src/main/java/alluxio/worker/block/BlockWorker.java

```diff
@@ -14,12 +14,7 @@
 import alluxio.Configuration;
 import alluxio.Constants;
 import alluxio.Sessions;
-import alluxio.exception.AlluxioException;
-import alluxio.exception.BlockAlreadyExistsException;
-import alluxio.exception.BlockDoesNotExistException;
-import alluxio.exception.ConnectionFailedException;
-import alluxio.exception.InvalidWorkerStateException;
-import alluxio.exception.WorkerOutOfSpaceException;
+import alluxio.exception.*;
```

Comment:
Please follow the import orders in Coding Style [here](http://alluxio.org/documentation/master/en/Contributing-to-Alluxio.html)

- For Eclipse to organize your imports correctly, configure “Organize Imports” to look like [this](http://alluxio.org/documentation/master/resources/eclipse_imports.png)
- For IntellJ, configure in Preferences->Code Style->Java->Imports->Import Layout according to this [order](http://alluxio.org/documentation/master/resources/intellij_imports.png)